components/card/stories/card.test.js

```diff
@@ -9,28 +9,22 @@ export const CardGroup = Variants({
 		},
 		{
 			testHeading: "No image",
-			title: "Card title",
-			description: "Optional description that should be one or two lines",
 			footer: undefined,
 			image: undefined,
 		},
 		{
 			testHeading: "Horizontal",
-			title: "Card title",
 			description: "jpg",
 			showAsset: "file",
-			isQuiet: false,
```

Comment:
All the changes in here aren't necessary- we just don't need to redefine a bunch of stuff. This was sort of an extension of some of the removals in `card.stories.js`.

Reply:
Great updates! Love an optimization. ❤️